src/App.tsx

```diff
@@ -1,24 +1,131 @@
-/* eslint-disable max-len */
+/* eslint-disable @typescript-eslint/no-unused-expressions */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
+import { Header } from './components/header/Header';
+import { List } from './components/list/List';
+import { Footer } from './components/footer/Footer';
 import { UserWarning } from './UserWarning';
+import { Warnings } from './components/warnings/Warnings';
+import { deleteTodo, getTodos, updateTodo } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoFilter } from './types/TodoFilter';
+import { ErrorMessage } from './types/ErrorMessage';
 
-const USER_ID = 0;
+const USER_ID = 10514;
+
+const { All, Active, Completed } = TodoFilter;
 
 export const App: React.FC = () => {
+  const [todoList, setTodoList] = useState<Todo[]>([]);
+  const [filterBy, setFilterBy] = useState(All);
+  const [error, setError] = useState('');
+  const [processing, setProcessing] = useState<number[]>([]);
+
+  const loadTodos = () => {
+    getTodos(USER_ID)
+      .then((todos) => setTodoList(todos))
+      .catch(() => setError(ErrorMessage.Get));
+  };
+
+  useEffect(() => loadTodos(), []);
+
+  useEffect(() => {
+    setTimeout(() => setError(''), 3000);
+  }, [error]);
+
+  const completedTodos = useMemo(() => todoList
+    .filter(todo => todo.completed).length, [todoList]);
+
+  const leftItems = todoList.length - completedTodos;
+
   if (!USER_ID) {
     return <UserWarning />;
   }
 
+  function filterList() {
```

Comment:
It's better to use arrow functions. It makes React code more concise and easier to read